kitsune/wiki/jinja2/wiki/base.html

```diff
@@ -5,3 +5,6 @@
 {% if product %}
   {% set search_params = {'product': product.slug} %}
 {% endif %}
+{% if not ga_content_group %}
+  {% set ga_content_group = "kb-other" %}
```

Comment:
I was thinking of this as a default value for `ga_content_group` for any future wiki page that didn't set it, so at least the new page is placed within a KB bucket rather than being placed within the `(unset)` content group. However, I'm fine with removing it too.